src/tbp/monty/frameworks/models/evidence_unsupervised_inference_matching.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+from tbp.monty.frameworks.models.evidence_matching import MontyForEvidenceGraphMatching
+
+
+class MontyForUnsupervisedEvidenceGraphMatching(MontyForEvidenceGraphMatching):
```

Comment:
The name of the class could be easily misunderstood as being an unsupervised learning experiment. How about something like `MontyWithoutReset` or `MontyWithoutEpisodeBorders`?